pylearn2/sandbox/nlp/models/tests/composite.yaml

```diff
@@ -0,0 +1,78 @@
+!obj:pylearn2.train.Train {
+    dataset: &train !obj:pylearn2.datasets.vector_spaces_dataset.VectorSpacesDataset {
+        data: !obj:pylearn2.sandbox.nlp.models.as_tuple { list_: [
```

Comment:
I think it may be better to fix `VectorSpacesDataset` so it accepts a list as `data`.